### ISSUE TITLE
Need a different icon for lumber

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#757]
Separate from boards

**Comment by keegan-o** [comment]
adding a placeholder for logs--

We don't actually need a new icon for lumber, we need a new icon for logs.
It should be:

Logs
![image](https://cloud.githubusercontent.com/assets/14983165/19977541/27a68f3a-a1b1-11e6-86fd-d981474f3982.png)

Lumber
![image](https://cloud.githubusercontent.com/assets/14983165/19977578/47f62c00-a1b1-11e6-8b68-86fede503290.png)

Boards
![image](https://cloud.githubusercontent.com/assets/14983165/19977551/33c00a30-a1b1-11e6-9fcc-71fd93da343a.png)

Basically Logs should look like freshly cut trees, Lumber should look like processed logs, and boards should look like boards.
